index.html

```diff
@@ -10,21 +10,30 @@
 </head>
 
 <body>
-    <header class="white">
-        <a class="logo" href="#">My logo</a>
-        <nav class="nav-bar">
-            <div class="bars"><img src="images/menu.png" alt="menu navbar" /></div>
-            <div class="desk-menu">
-                <ul>
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul></div>
-        </nav>
+    <header class="menu-animation">
+        <div class="container">
+            <div class="logoforJS">
+               <h2>My logo</h2>
+            </div> 
+
+            <div class="nav-toggle" id="navToggle">
+                <img src="./images/menu.png" alt="hamburger-menu"/>
+            </div>        
+           
+            <nav class="desktop-nav">            
+                <ul class="links">
+                    <li class="close">X</li>                    
+                    <li class="nav-items"><a href="#porto">Portfolio</a></li>
+                    <li class="nav-items"><a href="#aboutMe">About</a></li>
+                    <li class="nav-items"><a href="#contact-me">Contact</a></li>

```

Comment:
- [ ] Great job so far by implementing the mobile-menu. Everything work as expected but it seems that while implementing it, something changed in your desktop version. For now, the navigation section disappears on your desktop version. means when you declared in your css `display: none` for only the hamburger button, whole the navigation disappears. Kindly, fix this issue so that we can see the navigation menu on the desktop version as well as on the mobile version.

| Your project | Figma |
|---------------|--------|
|![image](https://user-images.githubusercontent.com/71874425/174066741-2593ce9d-17b3-4688-b822-fa189a3f9a42.png)|![image](https://user-images.githubusercontent.com/71874425/174066972-485c854d-e473-489b-8833-ad850a211ede.png)|